### ISSUE TITLE
generated flash and modal core components are not visible when calling open_browser

### ISSUE THREAD

**Issue by dvic** [phoenixframework/phoenix_live_view#2330]
In Phoenix 1.7, the flash and modals use `phx-mounted={@show && show_modal(@id)}`. Because of this, and the fact that `open_browser` strips away `script` tags, modals and flashes stay hidden when the browser is opened. Is this intended? It would be nice if the modals are visible when calling `open_browser`.

**Comment by chrismccord** [comment]
Open browser doesn't run the js, therefore it necessarily won't run the phx-mounted callbacks

**Comment by dvic** [comment]
Yes, I was aware of the reason for this behavior but I'm trying to understand the rationale. Is there a way to easily have the JS-only logic run in these test views? Or are there also other reasons for not wanting this in the first place?

In any case, I'll open up a PR to document the current behavior (open_browser/2 doesn't mention the removal of the script tag, for example).
